src/spdx_tools/spdx3/writer/json_ld/json_ld_converter.py

```diff
@@ -69,7 +69,7 @@ def _convert_to_json_ld_dict(element: Any, alt_creation_info=False, alt_hash=Fal
         elif attribute_value:
             if attribute_name == "_spdx_id":
                 attribute_name = "@id"
-            if attribute_name == "_from_element":
+            elif attribute_name == "_from_element":
```

Comment:
The underscores in front of the property names originate in our `dataclass_with_properties` implementation, which is currently under rework. Just saying that we possibly need to adapt this once we rebase.